doc/python/morphology.rst

```diff
@@ -522,6 +537,9 @@ constitute part of the CV boundary point set.
 
 .. _pyneuroml:
 
+NeuroML
```

Comment:
There's also an SWC section here somewhere. If this doesn't have a header, that probably doesn't either.

Reply:
It was all a bit messed up, I fixed the ordering and added headings to the file formats.

Reply:
I checked over the file, and the new ordering (input file formats at the bottom of the file) is more logical.